### PR TITLE
Write to gl_FragData instead of gl_FragColor for debug commands if MRT is used

### PR DESCRIPTION
**Pull request by austinEng**
Fixes (1) of #2448

### CODE REVIEW

**Comment by pjcozzi** [comment]
Since this is a user-facing change, please update [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md).  Look at example throughout that file for how to word this.

File: Source/Scene/Scene.js [hunk 1]
```diff
@@ -1433,22 +1433,38 @@ define([
...
             '    czm_Debug_main(); \n';
 
+        var i;
         if (scene.debugShowCommands) {
             if (!defined(command._debugColor)) {
                 command._debugColor = Color.fromRandom();
             }
             var c = command._debugColor;
-            newMain += '    gl_FragColor.rgb *= vec3(' + c.red + ', ' + c.green + ', ' + c.blue + '); \n';
+            if (length) {
```
Comment (pjcozzi) [inline Source/Scene/Scene.js]: Here and below, we are usually more explicit, e.g., `length > 0`

**Comment by pjcozzi** [comment]
@bagnell could you do the final review and merge?

**Comment by austinEng** [comment]
Updated

**Pull request merged**